docs/api/covidcast-signals/chng.md

```diff
@@ -185,6 +185,10 @@ Note that due to local differences in health record-keeping practices, estimates
 are not always comparable across locations. We are currently working on
 adjustments to correct this spatial bias.
 
+Indicator values for as-of dates before 2021-02-21 are merely estimates, as
```

Comment:
```suggestion
Indicator values for issue dates before 2021-02-21 are merely estimates, as
```

to be consistent with [our date documentation](https://cmu-delphi.github.io/delphi-epidata/api/covidcast_times.html)